ArchiSteamFarm/Bot.cs

```diff
@@ -4114,6 +4120,111 @@ internal sealed class Bot : IDisposable {
 			return responses.Count > 0 ? string.Join("", responses) : null;
 		}
 
+		private async Task<string> ResponsePrivacy(ulong steamID, string privacySettingsText) {
+			if ((steamID == 0) || string.IsNullOrEmpty(privacySettingsText)) {
+				ArchiLogger.LogNullError(nameof(steamID) + " || " + nameof(privacySettingsText));
+				return null;
+			}
+
+			if (!IsMaster(steamID)) {
+				return null;
+			}
+
+			if (!IsConnectedAndLoggedOn) {
+				return FormatBotResponse(Strings.BotNotConnected);
+			}
+
+			if (privacySettingsText.Length != 6) { // There are only 6 privacy settings
+				return FormatBotResponse(string.Format(Strings.ErrorIsInvalid, nameof(privacySettingsText)));
+			}
+
+			Steam.PrivacyResponse privacySettings = new Steam.PrivacyResponse {
```

Comment:
Move this below `for` parsing loop and add yourself constructor to solve `readonly` issue, like done in `Steam.TradeOffer`.